docs/source/coupling_traffic/introduction.rst

```diff
@@ -0,0 +1,149 @@
+Therefore, *SUMO-JuPedSim* is not intended as a standalone pe-destrian simulation tool; for more advanced pedestrian-specific configurations, *JuPedSim* alone remains the better choice.
+
+
+What is it for?
+===============
+
+The integration is particularly useful in scenarios where crowd and traffic dynamics interact, such as:
+
+*	Scenarios involving congestion-prone pedestrian facilities that influence urban traffic flow or vice versa.
+*	Crowded public spaces with connections to vehicular transport systems, e.g., train stations.
+*	Modeling intermodal arrival and departure traffic during major public events.
+
+In summary: The *SUMO-JuPedSim* integration becomes especially relevant when crowd dynamics play a significant role in traffic simulation scenarios.
+
+.. figure:: /_static/coupling/intro/Fig1_crossing.png
+    :width: 80%
+    :align: center
+    :alt: Simulation snapshot for a crossing scenario with pedestrians and cars
+
+    Simulation snapshot for a crossing scenario with multidirectional pedestrian streams that interact with cars.
+
+.. figure:: /_static/coupling/intro/Fig2_event.png
+    :width: 80%
+    :align: center
+    :alt: Simulation snapshot for the arrival process to a major event
+
+    Simulation snapshot for the arrival process to a major event considering different modes of transport (train, car, bus, by foot).
+
+
+Configuration Possibilities
+===========================
+
+Following functionalities for configuring a *JuPedSim* simulation in *SUMO* are available:
+
+*	Precise definition of a 2D **walkable area** that is accessible for *JuPedSim* agents including obstacles (such as barriers, trees and signs) via dxf-import tool or drawing in netedit
+*	Conversion of pedestrian facilities included in a *SUMO* network to a 2D walkable area
+*	Configuration of **model parameters** for the basic :class:`~jupedsim.models.CollisionFreeSpeedModel`
+*	Configuration or temporary adjustment of pedestrian speed in defined areas
+
+.. note::
+    Even though it is in theory possible to use other *JuPedSim* models, only the :class:`~jupedsim.models.CollisionFreeSpeedModel` has been tested extensively so far.
+
+*	Configuration of **sources** in which *JuPedSim* agents are spawned
+*	Configuration of coupled **transfer** points where agents switch from a vehicle (bus, cars or train) to walking or vice versa
+*	Configuration of flexible **routing** (journeys) for agents consisting of intermediate waypoints, distribution waypoints and exits
+*	Configuration of **vanishing zones** at exits to model ticket control at venues according to the capacity of an entrance system
+*	**Interaction of cars** and pedestrians in an ordered manner (stopping by management measures)
+
+
+
+Technical Background
+====================
+
+*SUMO* (**S**\ imulation of **U**\ rban **Mo**\ bility) is a collection of programs for modeling traffic flow microscopically especially for urban or interurban scenarios.
```

Comment:
This intro feels like doppelt-gemoppelt. 
We already know this at this point. I would remove it completly or start with it the intro where you write about Sumo Eclipse.
(but again, I guess not too much because you can assume people reading this, have already heard one or two things about sumo)